mingw_setup/readme.txt

```diff
@@ -131,13 +131,46 @@ file in the next step.
     
     and install it
 
+7. Download git and tortoisegit 
+_______________
+Download git from here:
 
-7. Get the assets
+https://git-scm.com/download/
+
+and install it
+
+then Download tortoisegit from here:
+
+https://tortoisegit.org/download/
+
+and install it
+
+
+8.Get the project
+_______________
+now right click in the place you want to put your project into for me i prefer to make folder called building thrive 
+(put it in the place you like)and inside it Right-click and choose git clone in URL write:
+
+https://github.com/Revolutionary-Games/Thrive.git
+
+and in Directory the place you want to put the project files in and then \new_name_for_file_here 
+(ex. D:\bulding thrive\Thrive)
+
+then click ok
+
+9.adding the project to system path
```

Comment:
It is not really necessary to add the project or anything to system path (you can verify this by removing the stuff you added to system path and rebuilding) so we should remove this point 9